label_studio/io_storages/gcs/form_layout.yml

```diff
@@ -15,7 +15,7 @@ gcs_credentials: &gcs_credentials
   - type: password
     name: google_application_credentials
     label: Google Application Credentials
-    placeholder: Copy and paste here the content of credentials.json
+    placeholder: Paste here the content of credentials.json OR leave blank to use ADC
```

Comment:
This might be clearer phrasing:
"Paste the contents of credentials.json here OR leave blank to use ADC"